File: hanwook/week_3/subak.js
function solution(n) {
    var answer = '';
    
    var subak = new Array(n).fill("수")
    
    for(var i = 1; i < n; i+=2) {
        subak[i] = "박"
    }
    
    answer = subak.join('');
    
    return answer;
}

File: dohee/week_3/Subak.java
package week_3;

public class Subak {
    public static String solution(int a) {
        String answer = "";

        for(int i=0; i<a/2; i++) {
            answer +="수박";
        }

        if(a % 2 !=0) {

            answer+="수";

        }

        return answer;
    }

    public static void main(String[] args) {
        solution(3);
    }
}
